### PR TITLE
Remove screens for community packages

### LINKED ISSUE #150: Remove all community packages Screen
RNTester contains a lot of screens for the component/APIs which have been moved outside the core. These need to be removed.

**Note**: Some of the community packages are also used in other screens. Those are not part of this issue. They will be removed when those screens will be reworked.

### PR DESCRIPTION
**Pull request by anku255**
## Summary

fixes #150 

## Changelog

Removed the following screens - 

- AsyncStorage
- Clipboard 
- DatePickerAndroid
- Picker
- PickerIOS
- ProgressBarAndroid
- ProgressViewIOS
- SegmentedControl
- SegmentedControlIOS
- PushNotificationIOS
- Slider

## Test Plan

Run the app on android and iOS. I have tested it on android and it works.

**Note**: I could not test it on iOS because of the similar issues @stealthanthrax  was facing.

### CODE REVIEW

**Comment by anku255** [comment]
@AnshG714 Can you please pull this locally and make sure it works on iOS? You are the only one with working iOS setup right now 😅

File: packages/RNTester/package.json [hunk 1]
```diff
@@ -12,11 +12,9 @@
   "dependencies": {
     "@react-native-community/async-storage": "^1.11.0",
     "@react-native-community/cli-platform-android": "^4.10.1",
-    "@react-native-community/datetimepicker": "^2.4.2",
     "@react-native-community/picker": "^1.6.3",
     "@react-native-community/progress-bar-android": "^1.0.3",
     "@react-native-community/progress-view": "^1.0.3",
-    "@react-native-community/push-notification-ios": "^1.2.2",
```
Comment (YashKumarVerma) [inline packages/RNTester/package.json:19]: @AnshG714 i was referring to these actually

Comment (anku255) [reply packages/RNTester/package.json:19]: @YashKumarVerma These are not used anywhere else, hence I removed them.

**Comment by anku255** [comment]
> @anku255 Do you think we should hold off on this? I was just thinking whenever the community tester app is being built we could copy over all the existing code there. And since we don't know if the community tester app is within our scope yet, deleting these screens might not be necessary right now? The changes look fine, this is more of a logistical consideration. What do you think?

These screens will be removed regardless of the Community Tester is built or not. But still, if the team feels this should be done towards the end, then I have no issues holding it off for now.

**Pull request closed without merging**